xcodeproject/xcodeproj/xcodeproj.go

```diff
@@ -106,6 +106,24 @@ func (p XcodeProj) TargetInfoplistPath(target, configuration string) (string, er
 	return p.buildSettingsFilePath(target, configuration, "INFOPLIST_FILE")
 }
 
+// DependentTargetsOfTarget returns with all dependencies of a given target, including the transitive dependencies.
+// The returned list contains each target only once, using the target ID for uniqueness.
+func (p XcodeProj) DependentTargetsOfTarget(target Target) []Target {
+	var dependentTargets []Target
+	for _, dependency := range target.Dependencies {
+		childTarget, ok := p.Proj.Target(dependency.TargetID)
+		if !ok {
+			log.Warnf("couldn't find dependency %s of target %s (%s), skipping", dependency.TargetID, target.Name, target.ID)
```

Comment:
You probably want to add a continue line after the warning.

Reply:
Whoops, fixed!